core/src/wallet/tezos/api_impl/TezosLikeTransactionApi.cpp

```diff
@@ -461,7 +461,7 @@ namespace ledger {
                             pub_key.c_str(), static_cast<SizeType>(pub_key.length()), allocator);
                         revealOp.AddMember("public_key", vString, allocator);
 
-                        static const auto fee = "257000";
+                        static const auto fee = "10000";
```

Comment:
Is there some doc link we could add for this magic value?

Reply:
https://tezos.stackexchange.com/a/787

Reply:
Not really a doc though ^^ And the source ends on a 404

Reply:
This code aimed to produce a dry-run-ready payload for [estimating gas limit](https://github.com/LedgerHQ/lib-ledger-core/blob/dca4496b1de21fecd303ce8dafc05d94088f9fd1/core/src/wallet/tezos/explorers/TezosLikeBlockchainExplorer.cpp#L223) during a transaction crafting (unsigned tx). The value should be low enough to avoid triggering a balance error from the node but, after some empirical tests, this specific value does not change the result of the estimated gas limit.

Reply:
@YBadiss considering [this source of info](https://medium.com/chain-accelerator/how-to-use-tezos-rpcs-16c362f45d64) it appears that `fee` can be anything, I even have seen `0` as a value on some forums. So considering this, do you accept the changes?